index.js

```diff
@@ -4,9 +4,38 @@ const data = require('./data.js');  // Import of the data from './data.js'
 
 // Connection to the database "recipeApp"
 mongoose.connect('mongodb://localhost/recipeApp', { useNewUrlParser: true })
-  .then(() => {
-    console.log('Connected to Mongo!');
-  }).catch(err => {
-    console.error('Error connecting to mongo', err);
-  });
+  // .then(() => {
+  //   console.log('Connected to Mongo!');
+  // }).catch(err => {
+  //   console.error('Error connecting to mongo', err);
+  // });
 
+//Iteration 2
+// Recipe.create(
+//    { title : "Bobun",
+//     level: "Easy Peasy",
+//     ingredients : ["nem","salad"],
+//     cuisine: "Asian",
+//     dishType : "Dish",
+//     duration : 30,
+//     creator: "Chef Duc",
+//     created : new Date(),}
+//   )
```

Comment:
at the end of your creation, you can add a promise (then / catch) so you can have feedback if the creation is done ! It is not mandatory but it can help you :)

`.then(recipeDoc => {
    console.log("New recipe CREATED correctly ! Yummmm.", recipeDoc);
  })
  .catch(err => {
    console.log("Recipe's creation failed...", err);
  });`

Reply:
Thanks, very helpful your corrections. I'll have a look